content/applications/general/product_images.rst

```diff
@@ -0,0 +1,135 @@
+
+#. Enable the API.
+
+   .. image:: product_images/gcp-custom-search-api.png
+      :align: center
+      :alt: "Custom Search API" tile with Enable button highlighted on Google Cloud Platform
+
+.. _product_images/google-pse-dashboard:
+
+Google Programmable Search dashboard
+------------------------------------
+
+#. Go to `Google Programmable Search Engine <https://programmablesearchengine.google.com/>`_ and
+   click on **Get Started**. Log in with your Google account.
+
+   .. image:: product_images/google-pse.png
+      :align: center
+      :alt: Google Programmable Search Engine page with the **Get Started** button on the up-right
+            of the page
+
+#. Fill the language and the name of the search engine. Give it an explicit name
+   (e.g. Odoo Images).
+
+   .. note::
+      Google doesn't allow to create a search engine without having entered at least one specific
+      site to search on. You can put any website (e.g. www.google.com) for this step, we will
+      remove it later.
+
+#. Validate the form by clicking on **Create**. Then, go to the edition mode of the search engine
+   that you created (either by clicking on **Control Panel** on the confirmation page or by
+   clicking on the name of your Search Engine on the Home page).
+
+#. In the **basics** tab, make sure to enable **Image search**, **SafeSearch** and 
+   **Search the entire web**.
+
+   .. note::
+      Once **Search the entire web** is enabled, you can safely delete the site that you put at the 
+      previous step.
+
+#. Save your **Search Engine Id**. You’ll need it for the next step in Odoo!
+
+.. _product_images/setup-in-odoo:
+
+Odoo
+----
+
+#. Go to :menuselection:`Settings --> General Settings --> Integrations`,
+   activate **Google Images** and save.
+
+#. Go back to :menuselection:`Settings --> General Settings--> Integrations`, enter your **API Key**
+   and **Search Engine ID** in **Google Images** settings and save again.
+
+.. _product_images/get-product-images:
+
+Automatically get your product images in Odoo
+=============================================
+
+The action to automatically get your product images in Odoo appears in any Products or Product
+Variants list view. Here is a step-by-step guide from the Inventory app.
+
+#. Go to the Products menu (:menuselection:`Products --> Products` or :menuselection:`Products --> 
+   Product Variants`) from any application that uses products like Inventory or Sale.
+
+#. On the list view, select the products that needs an image.
+
+   .. important::
+      Only the 10,000 first selected products or product variants will be processed.
+
+   .. note::
+      Only the products or product variants with a barcode and without an image will be processed.
+   
+   .. note::
+      If you select a product that has one or more variants from the Products view, each variant
+      matching the previous criteria will be processed.
```

Comment:
You can group notes, imo

```suggestion
   .. note::
      - Only the products or product variants with a barcode and without an image will be processed.
      - If you select a product that has one or more variants from the Products view, each variant
        matching the previous criteria will be processed.
```

Reply:
I suggested to split them when it was still a single paragraph but, as a bulleted list, it's fine too :+1: